metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/indexbuilder/ESIndexBuilder.java

```diff
@@ -220,6 +224,27 @@ public void buildIndex(ReindexConfig indexState) throws IOException {
     }
   }
 
+  public String reindexInPlaceAsync(String indexAlias, @Nullable QueryBuilder filterQuery, BatchWriteOperationsOptions options)
```

Comment:
I see, this function is generic. The issue is that the BatchWriteOperationsOptions says otherwise in that there is `timeseries` in the package. Since this function is generic, then the Options object should be as well and then the other functions in the Service classes above don't seem out of place (referencing `timeseries`) option.

Reply:
Can do, I'll move that. That seems better than taking in the batch size and timeout as parameters since we may want to do other tuning (or support other storages that have different parameters) in the future